### ISSUE TITLE
Code hints?

### ISSUE THREAD

**Issue by josebalius** [fdecampredon/brackets-flow#4]
Is there anyway to get autocomplete hints with this plugin?

**Comment by fdecampredon** [comment]
Normally code hints are implemented, do you get other features ? (quick edit error reporting) , did you forgot to add `/* @flow */` on top of your file ? 
You can also check if you got an error.

**Comment by josebalius** [comment]
@fdecampredon not sure what you mean? 

It is showing errors in the console at the bottom if I give the wrong type to an object but if I try typing I do not see any code hints coming up. Do I need to turn on a setting or another package perhaps?

**Comment by josebalius** [comment]
@fdecampredon hmm, it looks to break down whenever I have a `class` in the file. Do you know if brackets has issues with es6? (new to this editor)

Edit: nvm, it works with classes, I started a new file and im bringing over contents from another file line by line trying the code hints and it seems like if you have the following:

`console.log('\nMy debug line')` it breaks code hinting. if I remove the `\n` it works

**Comment by josebalius** [comment]
@fdecampredon so yeah, confirmed that it is the `\n` character breaking code hints. I also disabled your extension to see if it was a brackets issue, but it is isn't because code hints started to work even with the `\n` character present, as soon as I turned `brackets-flow` back on, it stopped working as long as I had a `\n` in the file. Any ideas what it could be?

**Comment by fdecampredon** [comment]
Yup I have some idea I guess it comes from the nodejs integration need to try it could you attach a sample to the issue ?

**Comment by josebalius** [comment]
@fdecampredon 

```
/* @flow */
type Test = {name: ''}

console.log('\nTest Log');

// Try hints here
```

**Comment by fdecampredon** [comment]
@josebalius yup the problem is pretty obvious
I try to escape things for unix command I guess I need to espace `\n` in a way but I don't know how :p will check when I have to do so thanks for the bug report

**Comment by josebalius** [comment]
@fdecampredon it's this line right: https://github.com/fdecampredon/brackets-flow/blob/master/node/flowDomain.js#L10 ? I can play with it and try to see if I can fix. I'll let you know. Apart from that, it works great! thank you
